modules/get_espe.py

```diff
@@ -79,12 +79,7 @@ def __init__(self, beam_ion: str, energy: float, theta: float,
         self.channel_width = ch
         self.fluence = fluence
         self.timeres = timeres
-        
-        if reference_density is None:
-            self.density = GlobalSettings().get_default_reference_density()
-        else:
-            self.density = reference_density
-        
```

Comment:
The user-defined global settings' reference density should be the default value (here and elsewhere). If it doesn't exist yet, initalize that from the `REFERENCE_DENSITY` (in **modules/global_settings.py**).

Reply:
What do you mean?

Reply:
Default reference density can be configured in Potku's settings, but its value isn't used in this branch. This is because `reference_density`  parameters have default values like this:

```python
reference_density: float = REFERENCE_DENSITY
```

When they should be more like this (I think?):

```python
reference_density: float = None
...
if reference_density is None:
    reference_density = GlobalSettings().get_default_reference_density()
```

Meanwhile, the `GlobalSettings().get_default_reference_density()` value stored in **potku.ini** (or some other file, I don't remember) should be initialized to `REFERENCE_DENSITY` if the global settings file doesn't exist yet. This is the case when there is no **Potku** folder in the user's home folder.